service/history/workflow/util.go

```diff
@@ -179,6 +181,62 @@ func FindAutoResetPoint(
 	return "", nil
 }
 
+func VerifyHistoryIsComplete(
```

Comment:
I think this doesn't belong in the `workflow` package. Maybe a separate util file in the `events` package?